app/views/automated_tests/student_interface.html.erb

```diff
@@ -31,6 +31,13 @@
     <% end %>
   </p>
 
+  <% unless @assignment.unlimited_tokens || @assignment.non_regenerating_tokens %>
```

Comment:
Please move this to immediately below the "Tokens regenerate" line (i.e., above "Tokens available")